psi4/src/CMakeLists.txt

```diff
@@ -64,14 +71,14 @@ install(TARGETS core
         EXPORT "psi4Targets"
         ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}${PYMOD_INSTALL_LIBDIR}/psi4
-        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+        RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}${PYMOD_INSTALL_LIBDIR}/psi4)
```

Comment:
`RUNTIME DESTINATION` is for executables, I thought. Why would they go to `lib/` or `site-packages/psi4`?

Reply:
On Windows, it is where `core.pyd` goes, while on Linux, it does not have an effect.

Reply:
@loriab this is the way to install libraries on Windows. @raimis note that `RUNTIME DESTINATION` does _have_ an effect on Linux, but only for targets created with `add_executable`.